File: rust/src/models/nango_config.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct NangoConfig {
    pub secret: String,
}

impl NangoConfig {
    pub fn new(secret: String) -> Self {
        Self { secret }
    }
}

File: javascript/src/models/nangoConfig.ts
// this file is @generated

export interface NangoConfig {
  secret: string;
}

export const NangoConfigSerializer = {
  _fromJsonObject(object: any): NangoConfig {
    return {
      secret: object["secret"],
    };
  },

  _toJsonObject(self: NangoConfig): any {
    return {
      secret: self.secret,
    };
  },
};
